data_steward/tools/clean_test_project.py

```diff
@@ -0,0 +1,107 @@
+# Python imports
+import argparse
+import logging
+import sys
+
+# Third party imports
+from googleapiclient.errors import HttpError
+
+# Project imports
+from utils import bq
```

Comment:
The lint check fails because `curation/data_steward/utils/__init__.py` doesn't exist.  Can you add that as part of your PR?

Reply:
Sure, I will add it